### ISSUE TITLE
LibretroDroid not calling FPSsync appropriately

### ISSUE THREAD

**Issue by tytydraco** [Swordfish90/LibretroDroid#46]
My FPS only syncs when the GLRetroView is first created and when I rotate to the horizontal orientation. When rotating to the portrait orientation, the FPSsync does not reinitialize.

On start / on landscape rotation I get this log: `Starting game with fps 60.000000 on a screen with refresh rate 90.000000. Using vsync: 0`

It is ABSENT on rotating to portrait, breaking my renderer until I rotate to landscape again.

**Comment by tytydraco** [comment]
Following up, this only occurs when immersive mode is enabled too soon after loading the game, and then a rotation ocurrs. This has been puzzling me for quite some time. Here's what I have tested (a fresh instance, reproducible for me):

- Apply immersive, then create GLRetroView: FAIL
- Create GLRetroView, then apply immersive: FAIL
- Create GLRetroView, sleep 5s, apply immserive: SUCCESS

In the failure cases, the speed is completely fine until I rotate the device (recreating the GLRetroView) while immersive mode is applied. In addition, the FPSsync logs are missing when rotating to portrait, indicating that LibretroDroid::resume was never called. Rotating back to horizontal in this glitched state works fine.

What solved my problem was to not apply immersive *at all*. With this setup, the glitch never occurs and the logs are normal.

Another interesting experiment was when I enabled windowTransparentNavigation and windowTransparentStatus, which appeared to cause this issue even without immersive mode being applied.

I have no idea where to begin debugging here. I've really tried everything, all I know is that the issue occurs only when the activity is restarted while immersive mode is enabled. If you have any hints as to where I should look in the LibretroDroid code, let me know. The only thing I can think of is maybe the lifecycle for the GLRetroView is removed before onPause can be called.
